pages/docs/concepts/asyncapi-document/extending-specification.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Extending Specification
+weight: 220
+---
+
+Extending the AsyncAPI specification is a technique that allows developers to include domain-specific or use-case-specific information not supported by the base specification. This extension capability provides customization, enabling APIs to accommodate unique details that would not otherwise fit within the confines of the standard AsyncAPI specification.
+
+The benefits of extending the AsyncAPI specification include enhanced specificity — delivering a clear encoding of necessary API details to clients, and consistency — allowing reusable components across the API. However, extensions should be used sparingly. Since they go beyond the Specification, they may not be universally understood.
```

Comment:
Be careful when using `clients` with AsyncAPI, cause there are none.

The only thing we have is `applications`, and an application is not directly related to the client/server concept, for example, both a client and server are seen as an application in AsyncAPI eyes.